server/start.go

```diff
@@ -8,7 +8,12 @@ import (
 	"runtime/pprof"
 	"time"
 
+	rosettacfg "github.com/cosmos/cosmos-sdk/server/rosetta/config"
+
+	"github.com/cosmos/cosmos-sdk/server/rosetta"
+
 	"github.com/spf13/cobra"
+	"github.com/tendermint/cosmos-rosetta-gateway/service"
```

Comment:
```suggestion
	rosettacfg "github.com/cosmos/cosmos-sdk/server/rosetta/config"

	"github.com/cosmos/cosmos-sdk/server/rosetta"

	"github.com/spf13/cobra"
	"github.com/tendermint/cosmos-rosetta-gateway/service"
```

This needs a `goimports -w` pass. Please use the githooks available in `contrib/`.